### PR TITLE
updates tests and interfaces to work with nemo@2.1.1

### PR DESCRIPTION
**Pull request by gabrielcsapo**
needed before merge 

- [x] https://github.com/paypal/nemo-locatex/pull/10

```
To https://github.com/gabrielcsapo/nemo-drivex.git
 + 4672e54...44aefb4 update-nemo-2.1.1 -> update-nemo-2.1.1
LM-NYC-00875272:nemo-drivex gacsapo$ grunt test
Running "jshint:files" (jshint) task
>> 2 files lint free.

Running "simplemocha:all" (simplemocha) task


  nemo-drivex @plugin@
    ✓ should be available
    @find@ method
      ✓ should exist
      ✓ should resolve to a web element
      ✓ should return a promise which is rejected (1054ms)
    @finds@ method
      ✓ should exist
      ✓ should resolve to an array of web elements
      ✓ should return a promise which is rejected (972ms)
    @present@ method
      ✓ should exist
    @visible@ method
      ✓ should exist
    @waitForElement@ method
      ✓ should exist
      ✓ should return true when an element exists
      ✓ should reject promise when element doesn't exist (1007ms)
    @waitForElementVisible@ method
      ✓ should be visible
      ✓ should return true when an element is visible (5423ms)
      ✓ should reject promise when element is not visible (7073ms)
    @oneDisplayed@ method
      ✓ should exist
    @allEnabled@ method
      ✓ should exist
    @allDisabled@ method
      ✓ should exist
    @allDisplayed@ method
      ✓ should exist
    @allHidden@ method
      ✓ should exist

  nemo-drivex setup method
    ✓ should return


  21 passing (19s)


Done, without errors.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,25 +7,25 @@
...
+    "async": "^1.5.2",
     "debug": "^2.1.1",
-    "nconf": "~0.6.7"
+    "nconf": "^0.8.4"
   },
   "main": "index.js",
   "registerAs": "drivex",
   "peerDependencies": {
-    "nemo": "^0.4.0",
+    "nemo": "^2.1.1",
```
Comment (grawk) [inline package.json]: i assume you need to update the nemo-locatex dependency before the CI will succeed

Comment (gabrielcsapo) [inline package.json]: made a note of this in the PR description 

```
needed before merge

- [ ] paypal/nemo-locatex#10
```

File: test/setup.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
+var nemo = {
 	"driver": true,
 	"wd": true
 };
 describe("nemo-drivex setup method", function () {
 	it("should return", function (done) {
-		NemoDriveX.setup({}, returnObj, function (err, config, returnObj) {
-			if (returnObj.drivex) {
-				//console.log("user", returnObj.user);
+		drivex.setup(nemo, function (err, nemo) {
```
Comment (grawk) [inline test/setup.js]: The setup function no longer passes back "nemo". Removing the second argument from the callback here should resolve.

**Comment by grawk** [comment]
:+1:

**Pull request merged**